### PR TITLE
feat: twilio

### PR DESCRIPTION
**Pull request by RafalWilinski**
Closes https://github.com/Stedi/registry/issues/57

### CODE REVIEW

File: src/providers/twilio.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      ],
+    };
+  }
+
+  async unbundle(bundle: OpenAPI3Schema): Promise<EntitySchema[]> {
+    const dereferenced = await openAPIParser.dereference(bundle.value as any);
+
+    const hasComponents = "components" in dereferenced;
+    if (!hasComponents) throw new Error("Expected components");
+
```
Comment (WojciechMatuszewskiStedi) [inline src/providers/twilio.ts:61]: I saw this condition in other providers as well. Would you mind elaborating on what "components" are in this context? I could not find any reference to this name anywhere while searching for it on JSONSchema website.

File: src/util/mock.ts [hunk 2]
```diff
@@ -148,6 +148,7 @@ const formatExamples: { [format: string]: string } = {
   date: "1970-01-01",
   uuid: "3fa85f64-5717-4562-b3fc-2c963f66afa6",
   "unix-time": "1647352387",
+  "date-time-rfc-2822": "01 Jun 2016 14:31:46 -0700",
```
Comment (WojciechMatuszewskiStedi) [inline src/util/mock.ts:151]: Would it be possible for us to get rid of `util` directory?

Maybe we could create `generate` directory with `generator.ts` and `readme.ts` in it.
The `mock.ts` file could be a single file, no need for putting it inside a directory.

WDYT?

File: src/providers/twilio.ts (hunk 1 above)
Comment (RafalWilinski) [reply src/providers/twilio.ts:61]: It's a section containing all the schemas in OpenAPI document: https://swagger.io/docs/specification/components

In this specific case, two lines above we're dereferencing an openapi spec and getting `components` from it.

Comment (WojciechMatuszewskiStedi) [reply src/providers/twilio.ts:61]: Ooh! I'm such a dummy, I've been searching on the wrong site (JSONSchema vs OpenAPI). Thank you for explanation.

It all makes sense now – thanks!

File: src/util/mock.ts (hunk 2 above)
Comment (RafalWilinski) [reply src/util/mock.ts:151]: https://github.com/Stedi/registry/issues/72

File: src/util/readme.ts [hunk 3]
```diff
@@ -18,7 +18,7 @@ function generateMapFromSchemaMarkdownButton({
   const mappingName = encodeURIComponent(
     `Mapping from ${capitalizeFirstLetter(providerName)}'s ${schemaName} schema`
   );
-  return `[![Map from this schema](/images/MapFromThisSchema.svg)](https://terminal.stedi.com/mappings/import?name=${mappingName}&source_json_schema=https://raw.githubusercontent.com/Stedi/registry/main/${target})`;
+  return `[![Map from this schema](/images/MapFromThisSchema.svg)](https://terminal.stedi.com/mappings/import?name=${mappingName}&referrer=registry-repo&source_json_schema=https://raw.githubusercontent.com/Stedi/registry/main/${target})`;
```
Comment (WojciechMatuszewskiStedi) [inline src/util/readme.ts:21]: Nit: Looking at the referer param in the starter-kit repo, I think there is no need to add `-repo` to the parameter value.

**Review by WojciechMatuszewskiStedi — APPROVED** [review]

**Pull request merged**